DevelopGitHub/GitHubExtension/GitHubExtension.Infrastructure.Extensions/Identity/CurrentProjectExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Security.Claims;
+using System.Security.Principal;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Infrastructure.Extensions.Identity
+{
+    public static class CurrentProjectExtensions
```

Comment:
Is it correct name for class holding extension methods for IPrincipal interface?